cluster-service/generate_helm_set_flags.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash 
+
+# The 'roleName' field in the ../config/config.yaml file is a comma separated list of role names.
+# This script separates them and generates `--set` flags for the `helm deploy` command.
+# Please note that since here, the role names are being separated by commas, the role name itself
+# cannot have a comma in it's name.
+
+genereate_set_flags() {
```

Comment:
Typo in the name of the function. genereate_set_flags -> generate_set_flags

Reply:
> I would also be explicit about it being helm related: generate_helm_set_flags

I have named the whole script to be this name, so I think the helm context can be gotten from the script name.

Reply:
> I have named the whole script to be this name, so I think the helm context can be gotten from the script name.

That is true for the context *within* the file but not outside of it. My comment here is coupled to the https://github.com/Azure/ARO-HCP/pull/1817/files#r2125972658 where you would call the function by name (or another one) from another file (the Makefile)

Reply:
Okay, I can agree with the function name to add `helm` in it. But I will still ask why not do all these in the script itself and keep the make file clean. https://github.com/Azure/ARO-HCP/pull/1817/files#r2126794479